cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/BuildNativeImageCommandTest.java

```diff
@@ -86,7 +86,7 @@ public void testBuildProjectWithAdditionalArgs() throws IOException {
         Path projectPath = this.testResources.resolve("nativeimageProject");
         System.setProperty(ProjectConstants.USER_DIR, projectPath.toString());
         BuildCommand buildCommand = new BuildCommand(projectPath, printStream, printStream, false,
-                false, true, "-H:Name=hoo");
+                false, true, "-o " + projectPath + "/hoo");
```

Comment:
Does this overwrite the output path? Is the test passing with this change?

Reply:
Noted. I'll have a deeper look.

Reply:
Just realised that these particular tests are not enabled. They apparently fails due to other various problem mentioned within comments. `testBuildProjectWithAdditionalArgs` test cannot accept `-o` because it is only allowed when building single ballerina files. `testBuildBalFileWithAdditionalArgs` however can accept `-o` as an argument through `new CommandLine(buildCommand).parseArgs(projectPath.toString(), "-o=hoo")`, but cannot test for the accuracy of the result because the test fails anyways. Would it be better to leave them unchanged?

Reply:
Ack, in that case lets keep the test as it is. And this should be fixed when we fix the test case

Reply:
Fixed in 46a85bb8f2d916d95459298a3a9f0b443d316ec0